frontend/src/views/camp/Invitation.vue

```diff
@@ -91,7 +107,7 @@ export default {
       return loginRoute(this.$route.fullPath)
     },
     profile () {
-      if (this.isLoggedIn !== true) {
+      if (this.states.INIT === this.dialogState) {
```

Comment:
Nachdem das camp geladen wurde, wird hier trotzdem this.api.get().profile() aufgerufen, was zu einem Fehler in der Konsole führt, falls man nicht eingeloggt ist.

Reply:
Wechsel auf berechung im backend implementiert